packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -202,6 +205,54 @@ const nodeErrorToAmplifyException = (err: NodeJS.ErrnoException): AmplifyExcepti
   );
 };
 
+type CommonExceptionEntry = {
+  messageIndicator: RegExp;
+  amplifyErrorType: AmplifyErrorType;
+  newMessage?: string;
+  resolution: string;
```

Comment:
should this just be an instance of an AmplifyError instead of the inputs to an AmplifyError?